frontend/src/pages/Integrations/IntegrationDetailPage/IntegrationDetailContentTabs/Configure.tsx

```diff
@@ -3,20 +3,36 @@ import './IntegrationDetailContentTabs.styles.scss';
 import { Button, Typography } from 'antd';
 import cx from 'classnames';
 import { MarkdownRenderer } from 'components/MarkdownRenderer/MarkdownRenderer';
-import { useState } from 'react';
+import useAnalytics from 'hooks/analytics/useAnalytics';
+import { INTEGRATION_TELEMETRY_EVENTS } from 'pages/Integrations/utils';
+import { useEffect, useState } from 'react';
 
 interface ConfigurationProps {
 	configuration: Array<{ title: string; instructions: string }>;
+	integrationId: string;
 }
 
 function Configure(props: ConfigurationProps): JSX.Element {
 	// TODO Mardown renderer support once instructions are ready
-	const { configuration } = props;
+	const { configuration, integrationId } = props;
 	const [selectedConfigStep, setSelectedConfigStep] = useState(0);
 
 	const handleMenuClick = (index: number): void => {
 		setSelectedConfigStep(index);
 	};
+
+	const { trackEvent } = useAnalytics();
+
+	useEffect(() => {
+		trackEvent(
+			INTEGRATION_TELEMETRY_EVENTS.INTEGRATIONS_DETAIL_CONFIGURE_INSTRUCTION,
```

Comment:
we added this event to  check if the user is aware by the current info that he needs to configure it manually rather than expecting it work out of the box. for selectedConfigStep we would need to add events for all the steps in the config. do we want to track that ?